styles.css

```diff
@@ -11,6 +15,64 @@ body {
   background: #fff;
 }
 
+/* Javascript menu */
+
+.mobile-menu {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  transform: translateY(-110%);
+  transition: transform 0.2s;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(10px);
+  list-style: none;
+}
+
+.menu-item {
+  align-items: center;
+  height: 24px;
+  margin-top: 50px;
+  left: 0;
+  right: 28px;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  list-style: none;
+}
+
+.menu-item,
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.hamburger {
+  z-index: 100;
+  background: #fff;
+  cursor: pointer;
+  border: none;
+}
+
+.menu-icon {
+  color: #6070ff;
+}
+
+.close-icon {
+  color: #fff;
+  display: none;
+}
```

Comment:
- [ ] Kindly make the close button visible as seen in the Figma design. I think the close(**X**) button visually gives the user a good experience since he's aware that clicking on it quit the menu options. We must not hover over it before seeing the icon.


| Yours |  Figma  |
|  --- |  ----  |
|  ![Screenshot from 2022-11-15 12-52-02](https://user-images.githubusercontent.com/63087491/201917949-d8d7ed57-ddf6-4293-b8e3-7c435395c547.png)  | ![Screenshot from 2022-11-15 13-17-20](https://user-images.githubusercontent.com/63087491/201917988-ab689b65-85fa-46ee-a0ab-f53e980978c5.png)  |